code/modules/nano/states/base.dm

```diff
@@ -40,19 +40,27 @@
   *
   * return NANO_state The state of the UI.
  **/
-/mob/proc/shared_nano_interaction()
+/mob/proc/shared_nano_interaction(atom/movable/src_object)
 	if (!client || stat) // Close NanoUIs if mindless or dead/unconcious.
 		return NANO_CLOSE
-	else if (restrained() || lying || stat || stunned || weakened) // Update NanoUIs if incapicitated but concious.
+	// Update NanoUIs if incapicitated but concious.
+	else if (restrained() || incapacitated() || lying || stunned || weakened)
```

Comment:
/mob/living/incapacitated() already checks for restrained(), stunned and weakened.